nativelib/src/main/resources/scala-native/libunwind/assembly.h

```diff
@@ -18,26 +18,26 @@
 
 #if defined(__powerpc64__)
 #define SEPARATOR ;
-#define PPC64_OFFS_SRR0   0
-#define PPC64_OFFS_CR     272
-#define PPC64_OFFS_XER    280
-#define PPC64_OFFS_LR     288
-#define PPC64_OFFS_CTR    296
+#define PPC64_OFFS_SRR0 0
+#define PPC64_OFFS_CR 272
+#define PPC64_OFFS_XER 280
+#define PPC64_OFFS_LR 288
+#define PPC64_OFFS_CTR 296
 #define PPC64_OFFS_VRSAVE 304
-#define PPC64_OFFS_FP     312
-#define PPC64_OFFS_V      824
+#define PPC64_OFFS_FP 312
+#define PPC64_OFFS_V 824
 #ifdef _ARCH_PWR8
 #define PPC64_HAS_VMX
 #endif
 #elif defined(__POWERPC__) || defined(__powerpc__) || defined(__ppc__)
 #define SEPARATOR @
 #elif defined(__arm64__)
-#define SEPARATOR %%
+#define SEPARATOR % %
```

Comment:
Is this correct?

Reply:
I had to fix the script first but now I can fix this. I need to revert that file and then do not format that file. I am not sure why everything works ok on `macOS` - it must not use the assembly.
```
// clang-format off
header in here
// clang-format on
```

Reply:
Ok, got that fixed. I had found that culprit but you found it faster.

Reply:
Builds and runs now.